### PR TITLE
Set up the tech building into a Tech Reconstructor.

### PR DESCRIPTION
**Pull request by GraionDilach**
This is a bit more advanced than the RA1 setup, allowing for stacking building effects up to 5 instances, inspired by YR Hospital and Machine Shop.

AI setup is intentionally missing for now because I think I'll replace the AI capture module.

### CODE REVIEW

File: mods/hv/rules/buildings.yaml [hunk 1]
```diff
@@ -1410,7 +1410,7 @@ TECHBUILDING:
 	Selectable:
 		Bounds: 40,48,0,-4
 	Tooltip:
-		Name: Tech Building
+		Name: Tech Rejuvenator
```
Comment (Mailaender) [inline mods/hv/rules/buildings.yaml:1413]: Rejuvenator is a bit odd. How are tanks rejuvenated. Maybe Titanium Armor Research Facility or something?

Comment (GraionDilach) [reply mods/hv/rules/buildings.yaml:1413]: That suggestion sounds convoluted - all our other names are simple. Reconstructor is my other idea.

Comment (Mailaender) [reply mods/hv/rules/buildings.yaml:1413]: Yes, that's better.

Comment (GraionDilach) [reply mods/hv/rules/buildings.yaml]: Updated with that proposal.

**Comment by GraionDilach** [comment]
Fixed the accidental omission of sequence renaming which broke the build.

**Pull request merged**